common/app/views/main.scala.html

```diff
@@ -27,7 +27,7 @@
 }
 
 <!DOCTYPE html>
-<html id="js-context" class="js-off is-not-modern id--signed-out" data-page-path="@request.path">
+<html id="js-context" class="js-off is-not-modern id--signed-out" lang="en" data-page-path="@request.path">
```

Comment:
I think we also maybe published something in Japanese. My feelings about document language checks was that maybe it's better to ignore the check on this basis so it doesn't flag up as an error in the prbuilds?